### PR TITLE
Set variable instead of hardcoded PATH value

### PR DESCRIPTION
**Pull request by anadahz**

### CODE REVIEW

File: lepidopter-fh/opt/ooni/remove-inc-reports.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-oonireport -f /etc/ooniprobe/ooniprobe.conf status |
```
Comment (hellais) [inline lepidopter-fh/opt/ooni/remove-inc-reports.sh:2]: Don't you need to source some config file somewhere?

File: lepidopter-fh/opt/ooni/upload-reports.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 source /etc/ooniprobe/ooniconfig.sh
```
Comment (hellais) [inline lepidopter-fh/opt/ooni/upload-reports.sh:2]: Like you do here.

File: lepidopter-fh/opt/ooni/remove-inc-reports.sh (hunk 1 above)
Comment (anadahz) [inline lepidopter-fh/opt/ooni/remove-inc-reports.sh:2]: @hellais Yes I missed that, pushing changes.
Thanks for reviewing the PR.

**Comment by anadahz** [comment]
[This](https://github.com/TheTorProject/lepidopter/commit/54534b14a18bf30511a415c55708803412927e8b) should be fixing this.

**Comment by bassosimone** [comment]
This can now be merged!

**Pull request merged**